integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -78,6 +75,16 @@
     AUX
   }
 
+  private String getModelImageName(String suffix) {
+    String currentDateTime = getDateAndTimeStamp();
```

Comment:
I thought that the test depends on the timestamp being the same throughout the entire run? If so, then currentDateTime should remain a global static final instead of being recalculated every time.

Reply:
changed